### DISCUSSION TITLE
2023-07-11

### DISCUSSION THREAD

**Discussion by nlom0218** [woowacourse-teams/2023-haru-study#17] [daily_meeting]
### 오늘의 할 일
- 준과의 면담
- 기획 확정 후에 와이어 프레임 스케치
- 깃허브 마일스톤, 프로젝트 설정

#### 공통

- api 명세 픽스
    - 세부정책 포함
- 브랜치 전략

#### 백엔드

- 기능목록 작성

#### 프론트엔드

- 디자인

### 언젠가 할 일

- `기획이 바뀐 이유` 포스팅 - 마코 담당

### 일정

- 노아: 5시 근로 
- 룩소: 근로 시간 미정

**Comment by jaehee329** [comment]
### 체크인
7점 - 비가 아주 많이 온다

### 체크아웃
7점 - 비가 아주 조금 그쳤다

**Comment by woosung1223** [comment]
### 체크인
7점 - 일어났는데 비가 엄청 많이와서 준비하는데 힘들었습니다.. 

### 체크아웃
8점 - 기획이 빨랑빨랑 됐고 UI도 이쁘게 나와서 1점 올랐읍니다

**Comment by nlom0218** [comment]
### 체크인

9점 - 비타민을 먹어야겠다. 비가 엄청 온다. 엄청 오는건 싫다(-1)

### 체크아웃

10점 - 새로운 기획안이 너무 마음에 든다. 디자인도 또한!

**Reply by woosung1223** [reply]
무슨 비타민 드시죠

**Reply by nlom0218** [reply]
오쏘물!!

**Comment by yeopto** [comment]
### 체크인

5점 - 비가 무슨 미친듯이 온다. 옷이 젖어서 -5

### 체크아웃

8점 - 일이 스무스하게 진행되기도 하고, 오늘 CI에 대해 테스트 해봐서 뿌듯하여 +3

**Comment by aak2075** [comment]
### 체크인

9점 - 어제 오기생겨서 달렸더니 조금 피곤하네요(-1)

### 체크아웃

10점 - 밥 먹고 와서 기분 좋네요 남은 시간 공부 열심히 하다 가야쥐~🚀

**Comment by woo-jk** [comment]
### 체크인
8점 - 일찍 나왔지만 차가 너무 막혀서 겨우 도착했다. 버스에서 조마조마해서 -2

### 체크아웃
10점 - 우리의 바뀐 기획이 마음에 든다!
